.github/workflows/library_go_tests.yml

```diff
@@ -0,0 +1,82 @@
+# This workflow performs tests in Go.
+name: Library Go tests
+
+on:
+  workflow_call:
+    inputs:
+      dafny:
+        description: 'The Dafny version to run'
+        required: true
+        type: string
+      regenerate-code:
+        description: "Regenerate code using smithy-dafny"
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  testGo:
+    strategy:
+      fail-fast: false
+      matrix:
+        library: [AwsEncryptionSDK]
```

Comment:
Add the test vectors as well here.

Reply:
TestVectors for Go aren't pushed yet. It's blocked on this PR - enabling it here will cause the CI to fail.